src/modules/tp/entities/tp.entity.ts

```diff
@@ -0,0 +1,26 @@
+import {Column, Entity, PrimaryGeneratedColumn} from "typeorm";
+import {BaseEntity} from "../../../commons";
+
+
+@Entity('trabajo')
+export class TpEntiy extends BaseEntity {
+
+    @PrimaryGeneratedColumn('increment',{name:'trabajo_id'})
+    id: number;
+
+    @Column({ name: 'jtp_id'})
+    jtp_id: number;
+
+    @Column({name: 'numero'})
+    number?: number;
+
+    @Column({ name: 'nombre'})
+    name: string;
+
+    @Column({ name: 'url'})
+    url: string;
+
+    @Column( { name: 'description'})
```

Comment:
Cambiar el name para que matchee con la db asi no se crean campos en la tabla
```suggestion
    @Column( { name: 'descripcion'})
```